.code-samples.meilisearch.yaml

```diff
@@ -657,3 +657,12 @@ document_guide_add_document_primary_key: |-
         "overview": "A boy is given the ability to become an adult superhero in times of need with a single magic word.",
         "release_date": "2019-03-23"
      }]' 
+tenant_token_guide_search_no_sdk_1: |-
+  curl \
+    -X POST 'http://127.0.0.1:7700/indexes/patient_medical_records/search' \
+    -H 'Authorization: Bearer [yourTenantToken]'
+tenant_token_guide_generate_sdk_1: |-
+tenant_token_guide_search_sdk_1: |-
```

Comment:
We need to use which one? `tenant_token_guide_search_sdk_1: |-` or `tenant_token_guide_sdk_search_1: |-` (they are different, here and in their usage)